### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by scott1y**
Hi Nick,
I finished the NLS work for the node-red-web-nodes.  Please review.  Thank you!

### CODE REVIEW

File: aws/aws.html [hunk 1]
```diff
@@ -16,20 +16,20 @@
 
 <script type="text/x-red" data-template-name="amazon s3 in">
     <div class="form-row">
-        <label for="node-input-aws"><i class="fa fa-user"></i> AWS</label>
```
Comment (knolleary) [inline aws/aws.html]: AWS is the name of the service and should not be NLS'd

File: box/locales/en-US/box.json [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
+    "box": {
+        "label": {
+            "box": "Box",
```
Comment (knolleary) [inline box/locales/en-US/box.json]: Box is the name of the service - should not be NLS'd

File: tfl/tfl-underground.html [hunk 3]
```diff
@@ -18,29 +18,29 @@
...
     <div class="form-row">
-        <label>&nbsp;<i class="fa fa-filter"></i> Get</label>
-        the status information for the
+        <label>&nbsp;<i class="fa fa-filter"></i> <span data-i18n="tfl-underground.label.get"></span></label>
+        <span data-i18n="tfl-underground.label.status"></span>
     </div>
     <div class="form-row">
             <label>&nbsp;</label>
             <select id="node-input-line" style="width:50% !important">
-                <option value="Bakerloo">Bakerloo</option>
```
Comment (knolleary) [inline tfl/tfl-underground.html]: All of these are proper names - not to be NLS'd

**Comment by knolleary** [comment]
General comment - no proper names should be NLS'd. Do not add them to the message catalog - have added inline comments for a couple of them, but this applies to all node names.

Will continue to review over the next few days - this will take a while.

**Comment by knolleary** [comment]
Note there's a travis build failure as well - https://travis-ci.org/node-red/node-red-web-nodes/builds/66286730#L342

**Comment by scott1y** [comment]
Hi Nick,  Thank you for your review!  I will change them accordingly.  By the way, just wondering that I feel some proper names would be translated.  For example, "London" is usually translated to Japanese and I guess to Chinese, too.  Not sure for "Bakerloo" though...

**Pull request merged**